pyproject.toml

```diff
@@ -165,3 +168,18 @@ cipaths = [ # In order to combine files from both the ubuntu and macos runners
     "/Users/runner/work/nnpdf/nnpdf",
     "/home/runner/work/nnpdf/nnpdf"
 ]
+
+[tool.repo-review]
+select = ["PY", "GH", "PP", "PC"]
+
+[tool.repo-review.ignore]
+"PY004" = "Using doc/ as folder"
```

Comment:
Can we not just change the folder name?

Reply:
We should but didn't want to deal with the possible conflicts with active PRs for just an `s`.